config/manager/distribution-configmap.yaml

```diff
@@ -8,18 +8,18 @@ metadata:
 # The distribution images are immutable and should not be updated
 immutable: true
 data:
-  ollama: docker.io/llamastack/distribution-ollama:latest
-  hf-endpoint: docker.io/llamastack/distribution-hf-endpoint:latest
-  hf-serverless: docker.io/llamastack/distribution-hf-serverless:latest
-  bedrock: docker.io/llamastack/distribution-bedrock:latest
-  cerebras: docker.io/llamastack/distribution-cerebras:latest
-  nvidia: docker.io/llamastack/distribution-nvidia:latest
-  open-benchmark: docker.io/llamastack/distribution-open-benchmark:latest
-  passthrough: docker.io/llamastack/distribution-passthrough:latest
-  remote-vllm: docker.io/llamastack/distribution-remote-vllm:latest
-  sambanova: docker.io/llamastack/distribution-sambanova:latest
-  tgi: docker.io/llamastack/distribution-tgi:latest
-  together: docker.io/llamastack/distribution-together:latest
-  vllm-gpu: docker.io/llamastack/distribution-vllm-gpu:latest
-  watsonx: docker.io/llamastack/distribution-watsonx:latest
-  fireworks: docker.io/llamastack/distribution-fireworks:latest
+  starter: docker.io/llamastack/distribution-starter:0.2.9
+  ollama: docker.io/llamastack/distribution-ollama:0.2.9
+  hf-endpoint: docker.io/llamastack/distribution-hf-endpoint:0.2.9
+  hf-serverless: docker.io/llamastack/distribution-hf-serverless:0.2.9
```

Comment:
these are already gone, right? 
same as cerebras, nvidia, open-benchmark, passthrough,watson,vllm-gpu

https://hub.docker.com/search?q=llamastack

Reply:
Yeah I will remove all the deprecated distributions